py/sight/data_structures.py

```diff
@@ -192,8 +193,24 @@ def log(
   elif isinstance(obj_to_log, pd.DataFrame):
     pandas_sight.log('pandas.DataFrame', obj_to_log, sight, frame)
   elif isinstance(obj_to_log, dict) or hasattr(obj_to_log, '__dict__'):
+    def sanitize_dict(d):
```

Comment:
Pull into separate function and add comments explaining why this is necessary for the avro format. We're changing the data so we need to document why so we can remove this code if the constraints from data formats change.

Reply:
got it!